packages/superset-ui-connection/types/external.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'fetch-mock';
```

Comment:
I think I need to add `@types/fetch-mock` as dev dependency to get tests to build.